src/store/conversations/conversations.reducers.ts

```diff
@@ -99,7 +100,7 @@ export const conversationsSlice = createSlice({
           };
         },
       );
-      state.conversations = state.conversations.concat(newConversations);
+      state.conversations = [...state.conversations, ...newConversations];
```

Comment:
Why do you need to change 'concat' to the 'spread operator' here?
From a performance point of view, concat is better.